Manito/Manito/Presentation/Scene/LoginScene/ViewController/LoginViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  LoginViewController.swift
+//  Manito
+//
+//  Created by Mingwan Choi on 2022/07/07.
+//
+
+import Combine
+import UIKit
+
+import SnapKit
+
+final class LoginViewController: UIViewController {
+
+    // MARK: - ui component
+
+    private let loginView: LoginView = LoginView()
+    
+    // MARK: - property
+    
+    private var cancellable = Set<AnyCancellable>()
+    private let viewModel: any BaseViewModelType
+    
+    // MARK: - init
+    
+    init(viewModel: any BaseViewModelType) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    deinit {
+        print("\(#file) is dead")
+    }
+    
+    // MARK: - life cycle
+    
+    override func loadView() {
+        self.view = self.loginView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.bindViewModel()
+    }
+
+    // MARK: - func
+    
+    private func bindViewModel() {
+        let output = self.transformedOutput()
+        self.bindOutputToViewModel(output)
+    }
+    
+    private func transformedOutput() -> LoginViewModel.Output? {
+        guard let viewModel = self.viewModel as? LoginViewModel else { return nil }
+        let input = LoginViewModel.Input(
+            appleSignButtonDidTap: self.loginView.appleSignButtonTapPublisher.eraseToAnyPublisher()
```

Comment:
p1;

appleSignButtonTapPublisher 는 이미 선언할때 AnyPublisher 로 변환하기 때문에 이 부분에서 eraseToAnyPublisher 가 없어도 될것이라 예상합니다. 
삭제부탁드립니다.

```suggestion
            appleSignButtonDidTap: self.loginView.appleSignButtonTapPublisher
```

Reply:
[046fc07](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/046fc07d03e9b0d0e859e1afc3ce8a141c234cd8)
반영하였습니다.